content/en/apps/guides/performance/telemetry.md

```diff
@@ -10,11 +10,11 @@ relatedContent: >
 
 _Introduced in v3.4.0_
 
-The app collects performance data on certain user actions which is then aggregated over each calendar month and replicated to the server. This can be used to evaluate the performance of the code and configuration and to evaluate where improvements can be made.
+The app collects performance data on certain user actions which is then aggregated each day and replicated to the server (versions prior to v3.4.12 do aggregate in a monthly basis instead of daily). This can be used to evaluate the performance of the code and configuration and to evaluate where improvements can be made.
```

Comment:
What is `v3.4.12` ?

Reply:
> (versions prior to v3.4.12 do aggregate in a monthly basis instead of daily)

For visibility, this should be highlighted as a note.

Reply:
> What is v3.4.12 ?

:man_facepalming: I made a mix of the version mentioned above v3.4.0 and the upcoming release v3.12.0.

> For visibility, this should be highlighted as a note.

Yes, it is a big change so worth to highlight.